src/ecs/systems/transform.rs

```diff
@@ -104,7 +105,7 @@ impl System<()> for TransformSystem {
                         // If the index is none then the parent is an orphan or dead
                         if let Some(parent_index) = self.indices.get(&child.parent()) {
```

Comment:
@ebkalderon I'm not sure, but when we use `ref` in this case, we have reference to reference.